nevergrad/benchmark/experiments.py

```diff
@@ -246,15 +246,19 @@ def paramultimodal(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
 
 # pylint: disable=redefined-outer-name
 @registry.register
-def yabbob(seed: tp.Optional[int] = None, parallel: bool = False, big: bool = False, noise: bool = False, hd: bool = False) -> tp.Iterator[Experiment]:
+def yabbob(seed: tp.Optional[int] = None, parallel: bool = False, big: bool = False, small: bool = False, noise: bool = False, hd: bool = False) -> tp.Iterator[Experiment]:
     """Yet Another Black-Box Optimization Benchmark.
     Related to, but without special effort for exactly sticking to, the BBOB/COCO dataset.
     """
     seedg = create_seed_generator(seed)
-    optims = ["NaiveTBPSA", "TBPSA", "NGO", "Shiva", "DiagonalCMA", "CMA", "PSO", "DE", "MiniDE", "QrDE", "MiniQrDE", "LhsDE", "OnePlusOne",
+    optims = ["NaiveTBPSA", "TBPSA", "DiagonalCMA", "CMA", "PSO", "DE", "MiniDE", "QrDE", "MiniQrDE", "LhsDE", "OnePlusOne",
               "TwoPointsDE", "OnePointDE", "AlmostRotationInvariantDE", "RotationInvariantDE", "CMandAS2", "CMandAS"]
     if not parallel:
-        optims += ["SQP", "Cobyla", "Powell", "chainCMASQP", "chainCMAPowell"]
+        optims += ["SQP", "Powell", "chainCMASQP", "chainCMAPowell"]
+    if not parallel and not small:
+        optims += ["Cobyla"]
+    if not small:
+        optims += ["NGO", "Shiva"]
```

Comment:
why remove them when it's small?

Reply:
Cobyla fails when the budget is too small ; I want to investigate it, but I won't have the time today, sorry.

Reply:
After offline discussion: Jeremy refers to small budget and sequential. I try to investigate this, maybe NGO can be improved there. I also investigate large budgets.

Reply:
@jrapin I want to investigate :)